profile.js

```diff
@@ -0,0 +1,62 @@
+const profileForm = document.getElementById("profile-form");
+const accountNameElem = document.getElementById("account-name");
+const accountNumberElem = document.getElementById("account-number");
+const changePinElem = document.getElementById("change-pin");
+const currentPinElem = document.getElementById("current-pin");
+const newPinElem = document.getElementById("new-pin");
+const confirmNewPinElem = document.getElementById("confirm-new-pin");
+const closeBtnElem = document.getElementById("close-btn");
+const updateBtnElem = document.getElementById("update-btn");
+// display of aaccount number
+accountNumberElem.value = currentUserAccountNumber;
+
+const allUsers = JSON.parse(localStorage.getItem("MB_USER_ACCOUNTS"));
```

Comment:
Don't we have a function for getting all users defined in common.js? Why do this again?